lib/board.rb

```diff
@@ -0,0 +1,25 @@
+class Board
+
+  attr_reader :cells
+  def initialize
+    @cells = {
+               "A1" => Cell.new("A1"),
+               "A2" => Cell.new("A2"),
+               "A3" => Cell.new("A3"),
```

Comment:
Maybe we can create a method to iterate the values at a later date!

Reply:
Sounds good -- I'm good to leave it like this for now.